libusb/libusb.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "libusb",
+    "config-opts": [ "--disable-static", "--disable-udev" ],
```

Comment:
Without udev, it won't have access to hwdb, and without it `lsusb` will be unable to print device names, which apps commonly rely on.

Reply:
Fixed it in https://github.com/flathub/shared-modules/pull/115. Thanks for the clarification.